### PR TITLE
Test generated PHP tests

### PR DESCRIPTION
**Pull request by andreamlin**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -165,6 +229,9 @@ workflows:
       - test-python:
           requires:
             - generate-clients
+      - test-php-7.0:
```
Comment (landrito) [inline .circleci/config.yml]: We should add add all language versions we care about to the other languages.

**Comment by andreamlin** [comment]
Added language versions.
PTAL

File: .circleci/config.yml [hunk 2]
```diff
@@ -18,6 +18,26 @@ anchor_test_ruby: &anchor_test_ruby
...
+  command: |
+    # Only run PHP API test if we have project credentials key
+    # or this API client doesn't require credentials.
+    if ! [ -z "$SYSTEM_TESTS_ENCRYPTION_KEY" ] || [ "$KEY_REQUIRED" = false ]
+    then
+      cd workspace
+      git clone \
+          https://github.com/GoogleCloudPlatform/google-cloud-php-${TEST_API}.git
+      cd google-cloud-php-${TEST_API}
+      cp -r ${PHP_CLIENTS_DIR}/google-cloud-pubsub-*/src \
```
Comment (michaelbausor) [inline .circleci/config.yml]: `google-cloud-pubsub-*`? Should this be `google-cloud-php-*`? Or `google-cloud-${TEST_API}-*`?

File: .circleci/config.yml [hunk 3]
```diff
@@ -149,6 +171,43 @@ jobs:
       - run:
           name: Test Logging.
           <<: *anchor_test_ruby
+  test-php-7.0:
```
Comment (michaelbausor) [inline .circleci/config.yml]: Why 7.0? If we had to pick only one version, then probably 7.2 is best.

Comment (andreamlin) [reply .circleci/config.yml]: Sure!

File: .circleci/config.yml [hunk 4]
```diff
@@ -143,6 +143,25 @@ anchor_run_ruby: &anchor_run_ruby
         name: Test Logging.
         <<: *anchor_test_ruby_client
 
+anchor_test_php: &anchor_test_php
+  command: |
+    # Only run PHP API test if we have project credentials key
+    # or this API client doesn't require credentials.
+    # We can remove this if-check when https://github.com/googleapis/gapic-generator/issues/2235
+    # is resolved and unblocks PHP unit tests without credentials.
+    if ! [ -e ${RUBY_GEN_DIR}/google-cloud-${TEST_API} ]
```
Comment (michaelbausor) [inline .circleci/config.yml]: RUBY_GEN_DIR?

Comment (andreamlin) [reply .circleci/config.yml]: ahh good eye!

**Pull request merged**